js/layout-javascript/news-feed-code.js

```diff
@@ -125,14 +125,6 @@ DynamicList.prototype.attachObservers = function() {
     }
   });
 
-  $(window).resize(function() {
```

Comment:
Why is the removal of this relevant to this PR?

Reply:
I believe it is shown here because I've done a git pull after I resolved conflicts.